src/fiskaltrust.Middleware.Interface.Client.Http.StrongName/version.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.3.4",
+  "version": "1.3.5",
```

Comment:
Should we stay with this versioning or go to 1.3.11?

Reply:
Good catch, changed to 1.3.11